app/src/main/java/app/nexd/android/ui/seeker/record/RecordFragment.kt

```diff
@@ -0,0 +1,24 @@
+package app.nexd.android.ui.seeker.record
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import app.nexd.android.R
+
+class RecordFragment : Fragment() {
```

Comment:
just to share as u might like it.
I recently learned that u can now pass the layout also in the constructor instead of overriding onCreateView.

Reply:
Thanks, yes, that would be easier for now and would work for a layout without data binding. But my understanding is that to get data binding to work in the view model we would still need to inflate the layout and return the view after using the generated data binding class to inflate.